pkgs/fc/ceph/fc/ceph/manage.py

```diff
@@ -420,18 +344,13 @@ def deactivate(self, flush=True):
 
         if flush:
             print(f"Flushing journal for OSD {self.id} ...")
-            run(
-                [
-                    "ceph-osd",
-                    "-i",
-                    str(self.id),
-                    "--flush-journal",
-                    "--osd-data",
-                    self.datadir,
-                    "--osd-journal",
-                    self._locate_journal_lv(),
-                ],
-                check=True,
+            run.ceph_osd(
+                # fmt: off
+                "-i", str(self.id),
+                "--flush-journal", "--osd-data",
```

Comment:
The grouping of arguments is a bit odd here. I'd expect argument name and argument value to be on the same line (e.g `"--osd-data", self.datadir,`).